components/monitoring/controllers/base/kube-prometheus-stack.yaml

```
@@ -0,0 +1,59 @@
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: HelmRepository
+metadata:
+  name: prometheus-community
+spec:
+  interval: 12h
+  type: oci
+  url: oci://ghcr.io/prometheus-community/charts
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: kube-prometheus-stack
+spec:
+  serviceAccountName: flux
+  interval: 1h
+  chart:
+    spec:
+      version: "69.6.0" # {"$imagepolicy": "flux-system:kube-prometheus-stack:tag"}
```

```suggestion
      version: "69.6.0" # {"$imagepolicy": "infra:kube-prometheus-stack:tag"}
```

You also need to create the update policies, see how I did it for cert-manager

---

Fixed, thanks!